WTWSMS/common/minor_titles/WtWSMS_imperial_legacy_minor_titles.txt

```diff
@@ -494,7 +544,14 @@ title_il_comes_sacrae_vestis = {
 		is_feudal = yes
 		has_romance_culture_trigger = yes
 		OR = {
-			has_landed_title = e_wre
+			AND = {
+				culture = roman
+				higher_real_tier_than = KING
+			}
+			AND = {
+				culture = roman
+				has_landed_title = e_wre
+			}
```

Comment:
In all titles in this file, I have explained why I disagree on them with you privately.